packages/server/test/PartnerDishes.test.ts

```diff
@@ -42,6 +46,72 @@ describe(`${PATH}`, () => {
     expect(res.body).toEqual(expect.objectContaining(reqBody));
   });
 
+  it('PUT /:id', async () => {
+    // Given
+    const agent = fixture.agent(Role.Partner, RESTAURANT_ID);
+    const dish = dishDto({ restaurant: RESTAURANT_ID });
+    const created = await fixture.db.dishModel.create(dish);
+    const id = created._id?.toString();
+    const reqBody = {
+      name: 'Danie 100',
+      mealType: ['śniadanie', 'lunch'],
+      description: 'Nowe ekstra danie',
+      price: 2400,
+      tags: ['wegańska', 'ostre'],
+      ingredients: [
+        {
+          name: 'cebula',
+          canBeExcluded: true,
+        },
+      ],
+      allergens: ['mleko'],
+      portionWeight: 100,
+      calories: {
+        per100g: 120,
+        perPortion: 10,
+      },
+      fats: {
+        per100g: 130,
+        perPortion: 20,
+      },
+      proteins: {
+        per100g: 140,
+        perPortion: 50,
+      },
+      carbohydrates: {
+        per100g: 100,
+        perPortion: 10,
+      },
+    };
+
+    // When;
+    const resp_put = await agent.put(`${PATH}/${id}`).send(reqBody);
+    // Then
+    expect(resp_put.status).toBe(HttpStatus.NO_CONTENT);
+
+    const resp_get = await agent.get(PATH);
+    expect(resp_get.body.data[0].name).toEqual(reqBody.name);
+    expect(resp_get.body.data[0].mealType).toEqual(reqBody.mealType);
+    expect(resp_get.body.data[0].description).toEqual(reqBody.description);
+    expect(resp_get.body.data[0].price).toEqual(reqBody.price);
+    expect(resp_get.body.data[0].tags).toEqual(reqBody.tags);
+    expect(resp_get.body.data[0].ingredients).toEqual(reqBody.ingredients);
+    expect(resp_get.body.data[0].allergens).toEqual(reqBody.allergens);
+    expect(resp_get.body.data[0].portionWeight).toEqual(reqBody.portionWeight);
+    expect(resp_get.body.data[0].fats).toEqual(reqBody.fats);
+    expect(resp_get.body.data[0].proteins).toEqual(reqBody.proteins);
+    expect(resp_get.body.data[0].carbohydrates).toEqual(reqBody.carbohydrates);
+
+    // When
+    let wrong_id = '111e1acfa3f9ad63169a0000';
+    if (id === wrong_id) {
+      wrong_id = '222e1acfa3f9ad63169a0000';
+    }
+    const wrong_token_resp = await agent.put(`${PATH}/${'111e1acfa3f9ad63169a0000'}`).send(reqBody);
```

Comment:
Nie wiem czy dobrze rozumiem, ale zamysł był taki, żeby zmienić hardcodowane `wrongId` na inne jeśli przypadkiem okaże się poprawne? (opierając się na poprawnym założeniu, że tylko jedno `id` na raz może być poprawne)

Jeśli tak to:
1. Po przypisaniu do `wrong_id` i tak podajesz do `.put`a literał a nie tą zmienną.
2. Możesz po prostu utworzyć nowe id (`new ObjectId().toString()`) i jest gwarantowane, że ono będzie inne od każdego poprzedniego, a zatem błędne. (technicznie rzecz biorąc mamy tą gwarancję tylko jeśli tworzymy mniej niż 16777216 id w jednej sekundzie w jednym wątku, ale to na 10000% nie ma miejsca)

Reply:
zapomniałem zmienić tej ścieżki na nową zmienną :/